requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+-r requirements.txt
+coverage
+flake8
```

Comment:
This file can be deleted, as dev dependencies should be managed by `tox`?

Reply:
Indeed, it is not needed.